docs/install.md

```diff
@@ -115,7 +115,14 @@ You can also build Nix from source by following the instructions at https://nixo
 
 	This will now create a nix-bitcoin node on the target machine.
 
-6. Nixops automatically creates an ssh key for use with `nixops ssh`. Access `bitcoin-node` through ssh in nix-shell with
+6. Resize the virtualbox disk
+        ```
+	./scripts/vbox-resize-disk1.sh
+	```
+
+	NixOps provides a virtualbox disk thats 50gb in size, but we need more than that to house the Bitcoin blockchain. Make sure to run this from within your nix-shell.
```

Comment:
nit: Can you mention the size it's changed to? Or even add an argument to `./scripts/vbox-resize-disk1.sh` with the size in GB

Reply:
done.